### PR TITLE
Fix campground fee

### LINKED ISSUE #24: Fix calculation of `campground_minimum_fee_yocto_near` when initializing the contract
Fix calculation of `campground_minimum_fee_yocto_near`. This should be the result

https://github.com/campgroundco/nft-contract/blob/ec3ed1cc3ad9f5f5d3696fd81e1778090e03a615/tests/create_serie-test.rs#L20

### LINKED ISSUE #25: Update TypeScript bindings and Markdown docs automatically when pushing to `main`
Currently the process of generating TypeScript bindings and Markdown docs is done manually. Add a new pipeline that can update the bindings and docs automatically without dev intervention.

@andreespirela how do you think we should tackle this?

### PR DESCRIPTION
**Pull request by acuarica**
Closes #24.
Closes #25.

### CODE REVIEW

**Comment by acuarica** [comment]
@andreespirela I also removed some warnings to have less clutter when compiling/testing. Moreover I rename some tests so that they can be more readable. I would like to hear your thoughts.

File: src/internal.rs [hunk 1]
```diff
@@ -38,9 +38,9 @@ pub(crate) fn calculate_fee(price: u128, campground_fee: u64, min_treasury: u128
     }
 }
 
-pub fn calculate_yocto_near(nears: f64) -> Balance {
-    (nears * (ONE_NEAR as f64)) as u128
-}
+// pub fn calculate_yocto_near(nears: u64) -> Balance {
```
Comment (andreespirela) [inline src/internal.rs:41]: Where did this go haha?

**Comment by andreespirela** [comment]
LGTM

File: src/internal.rs (hunk 1 above)
Comment (acuarica) [reply src/internal.rs:41]: Ha, I end up removing it. All call sites were just using a multiple of 10. The simplest solution was just to divide by 10 or 100 where needed.

**Pull request merged**